### DISCUSSION TITLE
FastHerbie forecast times from past or into future?

### DISCUSSION THREAD

**Discussion by andrewsheerin** [blaylockbk/Herbie#374] [Q&A] [answered]
Hello,

Before I entirely mess up my project, I want to confirm if forecast times are forecasting into the future from time step X, or are forecasting from the past to time step X. Given the example on the FastHerbie documentation page (screenshot attached).... Does Herbie HRRR model sfc product initialized 2022-Mar-01 00:00 UTC F02 give the forecast for 2022-Mar-01 00:02 or does it give 2022-Feb-28 22:00's two-hour forecast for 2022-Mar-01 00:00? 

Thanks,
Andrew

![image](https://github.com/user-attachments/assets/d8e2c4b4-d75e-47e6-b78d-ea48f3528155)

**Comment by williamhobbs** [comment, marked as answer]
Your first example is correct: 

> initialized 2022-Mar-01 00:00 UTC F02 give the forecast for 2022-Mar-01 00:02

**Reply by andrewsheerin** [reply]
Thanks!
